src/engine/test/assets/schemas/wazuh-logql-types.json

```diff
@@ -1,784 +1,1462 @@
-    "user.effective.group.name": "keyword",
-    "user.effective.hash": "keyword",
-    "user.effective.id": "keyword",
-    "user.effective.name": "keyword",
-    "user.effective.roles": "keyword",
-    "user.email": "keyword",
-    "user.full_name": "keyword",
-    "user.group.domain": "keyword",
-    "user.group.id": "keyword",
-    "user.group.name": "keyword",
-    "user.hash": "keyword",
-    "user.id": "keyword",
-    "user.name": "keyword",
-    "user.roles": "keyword",
-    "user.target.domain": "keyword",
-    "user.target.email": "keyword",
-    "user.target.full_name": "keyword",
-    "user.target.group.domain": "keyword",
-    "user.target.group.id": "keyword",
-    "user.target.group.name": "keyword",
-    "user.target.hash": "keyword",
-    "user.target.id": "keyword",
-    "user.target.name": "keyword",
-    "user.target.roles": "keyword",
-    "user_agent.device.name": "keyword",
-    "user_agent.name": "keyword",
-    "user_agent.original": "keyword",
-    "user_agent.os.family": "keyword",
-    "user_agent.os.full": "keyword",
-    "user_agent.os.kernel": "keyword",
-    "user_agent.os.name": "keyword",
-    "user_agent.os.platform": "keyword",
-    "user_agent.os.type": "keyword",
-    "user_agent.os.version": "keyword",
-    "user_agent.version": "keyword",
-    "vlan.id": "keyword",
-    "vlan.name": "keyword",
-    "vulnerability.category": "keyword",
-    "vulnerability.classification": "keyword",
-    "vulnerability.description": "keyword",
-    "vulnerability.enumeration": "keyword",
-    "vulnerability.id": "keyword",
-    "vulnerability.reference": "keyword",
-    "vulnerability.report_id": "keyword",
-    "vulnerability.scanner.vendor": "keyword",
-    "vulnerability.score.base": "number",
-    "vulnerability.score.environmental": "number",
-    "vulnerability.score.temporal": "number",
-    "vulnerability.score.version": "keyword",
-    "vulnerability.severity": "keyword",
-    "x509.alternative_names": "keyword",
-    "x509.issuer.common_name": "keyword",
-    "x509.issuer.country": "keyword",
-    "x509.issuer.distinguished_name": "keyword",
-    "x509.issuer.locality": "keyword",
-    "x509.issuer.organization": "keyword",
-    "x509.issuer.organizational_unit": "keyword",
-    "x509.issuer.state_or_province": "keyword",
-    "x509.not_after": "timestamp",
-    "x509.not_before": "timestamp",
-    "x509.public_key_algorithm": "keyword",
-    "x509.public_key_curve": "keyword",
-    "x509.public_key_exponent": "number",
-    "x509.public_key_size": "number",
-    "x509.serial_number": "keyword",
-    "x509.signature_algorithm": "keyword",
-    "x509.subject.common_name": "keyword",
-    "x509.subject.country": "keyword",
-    "x509.subject.distinguished_name": "keyword",
-    "x509.subject.locality": "keyword",
-    "x509.subject.organization": "keyword",
-    "x509.subject.organizational_unit": "keyword",
-    "x509.subject.state_or_province": "keyword",
-    "x509.version_number": "keyword"
+  "url": "url",
```

Comment:
which identation is the correct one? maybe if we respect the old one the PR will just show the added fields